src/script/conversation/ConversationRepository.ts

```diff
@@ -291,6 +291,7 @@ export class ConversationRepository {
 
     this.conversationRoleRepository = new ConversationRoleRepository(this.teamRepository, this.conversationService);
     this.leaveCall = noop;
+    this.scheduleMissingUsersAndConversationsMetadataRefresh();
```

Comment:
if this is supposed to be federation only, why isn't it conditional at all?

Reply:
there is no missing data so we'll iterate over an empty array and no request will be fired, but still we could add a isFederated check to avoid that as well but performance wise that's micro optimization

Reply:
is there no way to tell we are in a federated environment? i dont see why we should run a bunch of async calls every 3 hours if we dont need to.

Reply:
maybe `this.core.backendFeatures.isFederated`?

Reply:
https://github.com/wireapp/wire-webapp/pull/15158/commits/943806cda942c6839ec9f6ce0f8743c3f88e1a13